core/src/main/scala/org/apache/spark/rdd/WholeTextFileRDD.scala

```diff
@@ -51,7 +51,7 @@ private[spark] class WholeTextFileRDD(
       case _ =>
     }
     val jobContext = new JobContextImpl(conf, jobId)
-    inputFormat.setMinPartitions(jobContext, minPartitions)
+    inputFormat.setMinPartitions(sc, jobContext, minPartitions)
```

Comment:
You don't need to pass the context. You just need to pass the one value you're going to use from it.

Reply:
Yeah, thanks